lib/index.js

```diff
@@ -45,7 +48,11 @@ bedrock.events.on('bedrock-mongodb.ready', callback => {
       }, {
         collection: 'publicKey',
         fields: {owner: 1, pem: 1},
-        options: {unique: true, background: false}
+        options: {
+          background: false,
+          partialFilterExpression: {pem: {$exists: true}},
+          unique: true,
```

Comment:
Now that I think of it... we're probably using the uniqueness of this index in the code that adds keys on start up ... so we'll need to make an adjustment there if we make this index non-unique. But, again, I do think it's a problem that we can't shard if we have it ... and adding a simple existence check in the key adding code during startup is a better way forward.

Reply:
I've removed the unique flag.  It may already be the case that when we add keys at startup that we specify an `id` on the key, in which case that unique index will serve to prevent dups.